src/vm_cpu.rs

```diff
@@ -694,6 +734,12 @@ impl<'vcpu, 'pages, T: GuestStagePageTable> DerefMut for ActiveVmCpu<'vcpu, 'pag
     }
 }
 
+impl<'vcpu, 'pages, T: GuestStagePageTable> Drop for ActiveVmCpu<'vcpu, 'pages, T> {
```

Comment:
I don't think this will work. What happens if the active vCPU, for example, makes a call to `TvmRun`?

Reply:
Stopping counters unconditionally is really messy because it creates all sorts of side-effects during SBI calls. I don't know of the best solution, but we might result in a transition to a new guest. We can discuss it more tomorrow.